src/backend/marsha/core/tests/test_lti.py

```diff
@@ -217,6 +218,28 @@ def test_lti_passport_subdomain(self, *_):
         # We just have to make sure verification does not raise an exception
         lti.verify()
 
+    @override_settings(ALLOWED_HOSTS=[".example.com", "testserver"])
+    @mock.patch.object(LTIOAuthServer, "verify_request", return_value=True)
+    def test_lti_passport_allowed_host(self, *_):
+        """Launch request with a referer matching ALLOWED_HOSTS should succeed."""
```

Comment:
```suggestion
        """Launch request with a referer matching ALLOWED_HOSTS should succeed.
        
        This may happen with some browser that replace the referer header value by the referred-to
        and not the referred-from value (see https://bit.ly/35CvPXa).
        """
```